install/01_enabling_pg_stat_statements.mdx

```diff
@@ -21,12 +21,10 @@ On your database server, make sure that the extensions package is installed.
 
 Debian/Ubuntu:
 
-```
-# Replace 9.X with your installed Postgres version:
-sudo apt-get install postgresql-contrib-9.X
-```
+Debian/Ubuntu includes modules like `pg_stat_statements` in the main package,
+therefore no additional install is required.
```

Comment:
I double checked this but as Maciek mentions below places, this is no longer needed.

https://github.com/pganalyze/pganalyze-docs/pull/124
https://postgrespro.com/list/thread-id/2517774